python_ta/transforms/ExprWrapper.py

```diff
@@ -139,24 +160,44 @@ def apply_bool_op(self, op: str, values: Union[z3.ExprRef, List[z3.ExprRef]]) ->
 
         return value
 
-    def parse_unary_op(self, node: astroid.UnaryOp) -> z3.ExprRef:
+    def parse_unary_op(self, node: astroid.UnaryOp) -> ExprRef:
         """Convert an astroid UnaryOp node to a z3 expression."""
         left, op = node.operand, node.op
         left = self.reduce(left)
-
         return self.apply_unary_op(left, op)
 
-    def parse_bin_op(self, node: astroid.BinOp) -> z3.ExprRef:
+    def parse_bin_op(self, node: astroid.BinOp) -> ExprRef:
         """Convert an astroid BinOp node to a z3 expression."""
         left, op, right = node.left, node.op, node.right
         left = self.reduce(left)
         right = self.reduce(right)
 
         return self.apply_bin_op(left, op, right)
 
-    def parse_bool_op(self, node: astroid.BoolOp) -> z3.ExprRef:
+    def parse_bool_op(self, node: astroid.BoolOp) -> ExprRef:
         """Convert an astroid BoolOp node to a z3 expression."""
         op, values = node.op, node.values
         values = [self.reduce(x) for x in values]
-
         return self.apply_bool_op(op, values)
+
+    def parse_arguments(self, node: astroid.Arguments) -> Dict[str, ExprRef]:
+        """Convert an astroid Arguments node's parameters to z3 variables."""
+        z3_vars = {}  # initialize mapping of z3 variables
+
+        annotations = node.annotations
+        arguments = node.args
+        for ann, arg in zip(annotations, arguments):
+            if ann is None:
+                continue
+            inferred = ann.inferred()
```

Comment:
I looked into the failing tests a bit more. Instead of using `.inferred`, use the `pylint.util.safe_infer` function, which returns `None` or a single inferred value (whereas `.inferred` returns a list).